explorer/client/src/components/ownedobjects/views/OwnedCoinView.tsx

```diff
@@ -0,0 +1,152 @@
+                                ({ Type }) => Type === typeV
+                            );
+                            return (
+                                <div
+                                    key={typeV}
+                                    className={
+                                        openedType === typeV
+                                            ? styles.openedgroup
+                                            : styles.closedgroup
+                                    }
+                                >
+                                    <div
+                                        onClick={
+                                            openedType === typeV
+                                                ? goBack
+                                                : openThisType(typeV)
+                                        }
+                                        className={styles.summary}
+                                    >
+                                        <div
+                                            className={
+                                                openedType === typeV
+                                                    ? styles.openicon
+                                                    : styles.closedicon
+                                            }
+                                        >
+                                            <ContentIcon />
+                                        </div>
+                                        <div>{handleCoinType(typeV)}</div>
+                                        <div>{subObjList.length}</div>
+                                        <div>
+                                            {subObjList[0]._isCoin &&
+                                            subObjList.every(
+                                                (el) => el.balance !== undefined
+                                            )
+                                                ? `${subObjList.reduce(
+                                                      (prev, current) =>
+                                                          prev.add(
+                                                              current.balance!
+                                                          ),
+                                                      Coin.getZero()
+                                                  )}`
+                                                : ''}
+                                        </div>
+                                        <div />
+                                    </div>
+                                    <div className={styles.openbody}>
+                                        {openedType === typeV &&
+                                            subObjList.map((subObj, index) => (
+                                                <React.Fragment
+                                                    key={`${typeV}${index}`}
```

Comment:
You can just use the `index` here, you don't need to prefix it with `typeV`

Reply:
Implemented